lib/input-view.js

```diff
@@ -23,7 +23,7 @@ export default class InputView extends View {
   initialize(prompt, onConfirmed) {
     this.prompt = prompt;
     this.onConfirmed = onConfirmed;
-    atom.commands.add(this.element, { 'core:confirm': this.confirm });
+    atom.commands.add(this.element, { 'core:confirm': () => this.confirm() });
```

Comment:
I'm going to refactor the input views soon. Since I noticed this quick fix half way through refactoring all of them.